docs/conceptual/node/tx-pool.md

```diff
@@ -2,12 +2,80 @@
+
+For `Frame` based runtimes, an account based system is used for transaction ordering. Every signed transaction needs to contain a nonce, which is incremented by 1 every time a new transaction is made. For example, the first transaction from a new account will have `nonce=0` and the second transaction will have `nonce=1`.
+
+At minimum, `Frame` transactions have a `provide` tag of `encode(sender ++ nonce)` and a `require` tag of `encode(sender ++ (nonce -1)) if nonce > 1`. Transactions do not `require` anything if `nonce=0`. As a result, all transactions coming from a single sender will form a sequence in which they should be included.
+
+Substrate supports multiple `provide` and `require` tags, so custom runtimes can create alternate dependency (ordering) schemes.
+
+## Transaction Priority
+
+Transaction `priority` in the `ValidTransaction` [struct](https://substrate.dev/rustdocs/master/sp_runtime/transaction_validity/struct.ValidTransaction.html), helps determine the ordering of transactions that have their dependencies satisfied. The higher the priority number, the more important the transaction is.
+
+`Priority` helps define the linear ordering of a graph in the case of one transaction unlocking multiple dependent transactions. For example, if we have two (or more) transactions that have their dependencies satisfied, then we use `priority` to choose the order for them.
+
+
+For `Frame` based runtimes, `priority` is defined as the `fee` that the transaction is going to pay. For example:
+
+  * If we receive 2 transactions from *different* senders (with `nonce=0`), we use `priority` to determine which transaction is more important and included in the block first.
+
+  * If we receive 2 transactions from the *same* sender with an identical `nonce`, only one transaction can be included on-chain. We use `priority` to choose the transaction with a higher `fee` to store in the transaction pool.
+
+Note that the pool does not know about fees, accounts or signatures - it only deals with the validity of the transaction and the abstract notion of `priority`, `requires` and `provides` parameters. All other details are defined by the runtime via the `validate transction` implementation.
+
+## Transaction Lifecycle
+
+There are two paths a transaction can follow:
+
+1. **A block is received from the network**
+
+  The block is run through `execute_block`([code](https://github.com/paritytech/substrate/blob/20a9b15cdbed4bf962a4447e8bfb812f766f2fbc/frame/executive/src/lib.rs#L196)), and the entire block either succeeds or fails.
+
+2. **A block is produced by our own node**
```

Comment:
I would reverse the order of these (so make this (1)). The lead-in says there are two paths a **transaction** can take and then you start talking about executing **blocks.**

Ditto about removing code in this section. Maybe `BlockBuilder` should link to the reference docs on that.